src/testdrive.F90

```diff
@@ -306,9 +312,150 @@ end subroutine collect_interface
 
   character(len=*), parameter :: fmt = '(1x, *(1x, a))'
 
-
-contains
-
+  contains
+  
+  !> Open JUnit.xml file for CLI output of test results.
+  subroutine junitxml_open_file()
+  
+    implicit none
```

Comment:
These `implicit none` are not needed, as it is inside a module that already states it.

Reply:
Accepted.

Reply:
@jvdp1: I am not sure about the code of conduct.
Do you resolve this conversation? Or is it okay, if I do it?

Reply:
Either way is fine.